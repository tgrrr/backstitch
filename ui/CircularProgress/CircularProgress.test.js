import React from 'react'
import renderer from 'react-test-renderer'
import { CircularProgress } from '.'

it('CircularProgress: default', () => {
    const component = renderer.create(
        <CircularProgress />
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
