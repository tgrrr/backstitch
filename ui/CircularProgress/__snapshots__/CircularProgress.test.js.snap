// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CircularProgress: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Circularprogress "
    data-testid="Circularprogress"
  >
    <div
      className="MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
      role="progressbar"
      style={
        Object {
          "height": 40,
          "width": 40,
        }
      }
    >
      <svg
        className="MuiCircularProgress-svg"
        viewBox="22 22 44 44"
      >
        <circle
          className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
          cx={44}
          cy={44}
          fill="none"
          r={20.2}
          strokeWidth={3.6}
          style={Object {}}
        />
      </svg>
    </div>
  </div>
</div>
`;
